api/v1beta1/openstackplaybookgenerator_types.go

```diff
@@ -31,8 +31,8 @@ type OpenStackPlaybookGeneratorSpec struct {
 	// Heat Settings
 	EphemeralHeatSettings OpenStackEphemeralHeatSpec `json:"ephemeralHeatSettings,omitempty"`
 	// +kubebuilder:default=false
-	// Debug
-	Debug bool `json:"debug,omitempty"`
+	// Interactive enables the user to rsh into the playbook generator pod for interactive debugging with the ephemeral heat instance
```

Comment:
Should we also include something indicating that a manual execution of the playbook generation will be required?  Anyhow, that aside, this PR looks ready to go

Reply:
sounds good, I added a new commit to add the suggestion